src/parser.py

```diff
@@ -245,6 +245,50 @@ def update_employees_mppe(data, employees):
     return employees
 
 
+def update_employees_mpac(data, employees):
+    if int(data.year) == 2021:
+        if int(data.month) == 1:
+            header = "mpac-01-2021"
+        elif int(data.month) == 2:
+            header = "mpac-02-2021"
+        elif int(data.month) == 3:
+            header = "mpac-03-2021"
+        elif int(data.month) == 4:
+            header = "mpac-04-2021"
+        elif int(data.month) == 5:
+            header = "mpac-05-2021"
+        elif int(data.month) == 6:
+            header = "mpac-06-2021"    
+        elif  int(data.month) >= 7:
+            header = "mpac-07-2021"
+    elif int(data.year) == 2022 and int(data.month) in [1,2]:
+        header = "mpac-01-2022"
+    elif (int(data.year) == 2022 and int(data.month) >= 3) or (int(data.year) == 2023 and int(data.month) <= 8):
+        header = "mpac-03-2022"
+    elif int(data.year) == 2023 and int(data.month) in [9,10,11]:
+        header = "mpac-09-2023"
+    else:
+        header = "mpac-12-2023"
```

Comment:
Sugestão: essa parte pode ser encapsulada em uma função mais simples:

`def get_mpac_header(year, month):
    if year == 2021:
        if month in range(1, 7):
            return f"mpac-0{month}-2021"
        elif month >= 7:
            return "mpac-07-2021"
    elif year == 2022:
        if month in [1, 2]:
            return "mpac-01-2022"
        elif month >= 3:
            return "mpac-03-2022"
    elif year == 2023:
        if month in range(1, 9):
            return "mpac-03-2022"
        elif month in [9, 10, 11]:
            return "mpac-09-2023"
    return "mpac-12-2023"`
    
    E depois você chamaria a função assim:
    
    `header = get_mpac_header(int(data.year), int(data.month))`